spec/time_spec.rb

```diff
@@ -0,0 +1,58 @@
+# encoding: UTF-8
+
+require 'spec_helper'
+
+module ICU
+  describe TimeFormatting do
+    describe 'the TimeFormatting ' do
+      t0 = Time.at(1447338076) # in TZ=Europe/Prague Time.mktime(2015, 11, 12, 15, 21, 16)
+      t1 = Time.at(1445728517) # in TZ=Europe/Prague Time.mktime(2015, 10, 25, 01, 15, 17)
+      t2 = Time.at(1445735778) # in TZ=Europe/Prague Time.mktime(2015, 10, 25, 02, 16, 18)
+      t3 = Time.at(1445739439) # in TZ=Europe/Prague Time.mktime(2015, 10, 25, 03, 17, 19)
+      t4 = Time.at(1445743100) # in TZ=Europe/Prague Time.mktime(2015, 10, 25, 04, 18, 20)
+      t5 = Time.at(1427589321) # in TZ=Europe/Prague Time.mktime(2015, 03, 29, 01, 35, 21)
+      t6 = Time.at(1427592982) # in TZ=Europe/Prague Time.mktime(2015, 03, 29, 02, 36, 22)
+      t7 = Time.at(1427593043) # in TZ=Europe/Prague Time.mktime(2015, 03, 29, 03, 37, 23)
+      t8 = Time.at(1427596704) # in TZ=Europe/Prague Time.mktime(2015, 03, 29, 04, 38, 24)
+
+      f1 = TimeFormatting.create(:locale => 'cs_CZ', :zone => 'Europe/Prague', :date => :long , :time => :long)
```

Comment:
Since each formatter is only used for a single example, please create them inside the example block.